packages/appkit/exports/index.ts

```diff
@@ -2,6 +2,7 @@ import { CoreHelperUtil } from '@reown/appkit-core'
 import { AppKit } from '../src/client.js'
 import type { AppKitOptions } from '../src/utils/TypesUtil.js'
 import { PACKAGE_VERSION } from './constants.js'
+export { UniversalAdapterClient as UniversalAdapter } from '../src/universal-adapter/client.js'
```

Comment:
this is already exported from `@reown/appkit-adapters`

Reply:
I can't find this package in the repo, is it typo?

Reply:
Each adapter is a separate package ! 

If you check packages/adapters/wagmi => Here is `@reown/appkit-adapter-wagmi`

Reply:
you are right :( this one isnt, mb


Proper way would be to create a directoyry under `packages/adapters/` that would expose this just as any other adapter instead of exporting from here.